nulib/meadow
# Update to ueberauth-nusso v1.0.0

# Summary 
Update to `ueberauth-nusso` v1.0.0 to allow `ueberauth` to update to 0.7.x without causing an XSRF error

# Specific Changes in this PR
- Require `ueberauth-nusso ~> 1.0`
- Remove pin on `ueberauth v0.6.x`
 
# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test

1. `mix deps.get`

2. Look for the following in the output:
    ```
    Upgraded:
      ueberauth 0.6.3 => 0.7.0 (minor)
      ueberauth_nusso 0.3.0 => 1.0.0
    ```

    If you don't see the `ueberauth` update, check it by running `mix deps | grep ueberauth`. If it's still 0.6.x, run 

    ```
    mix deps.unlock ueberauth
    mix deps.update ueberauth
    ``` 

3. Once the versions are correct, run `mix phx.server` and make sure you can log in.

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
